src/Compilers/VisualBasic/Test/Emit/Emit/EditAndContinue/EditAndContinueStateMachineTests.vb

```diff
@@ -4456,26 +4456,6 @@ End Class
 Imports System
 Imports System.Threading.Tasks
 
-' TODO: the attribute shouldn't be needed (https://github.com/dotnet/roslyn/issues/9167)
-Namespace System.Runtime.CompilerServices
```

Comment:
Yes, it's [here](http://source.roslyn.io/#Roslyn.Compilers.CSharp.Emit.UnitTests/Emit/EditAndContinue/EditAndContinueStateMachineTests.cs,57b040310394020d). 
It didn't have to be modified though because the C# compiler didn't require the optional attribute (but the VB compiler did).